homeassistant/components/renault/coordinator.py

```diff
@@ -54,10 +57,24 @@ def __init__(
         )
         self.access_denied = False
         self.not_supported = False
+        self.assumed_state = False
+
         self._has_already_worked = False
+        self._hub = hub
 
     async def _async_update_data(self) -> T:
         """Fetch the latest data from the source."""
+
+        if self._hub.is_throttled():
```

Comment:
I think if it hasn't worked yet we should not return None
We should probably raise UpdateFailed instead

Reply:
```suggestion
        if self._hub.is_throttled():
            if not self._has_already_worked:
                raise UpdateFailed("Renault API throttled: init skipped")        
```